### ISSUE TITLE
Replace Qwen3-VL Conv3D patch embedding with Linear for NPU export

### ISSUE THREAD

**Issue by mhs4670go** [Samsung/TICO#861]
## Background

Qwen3-VL uses a `Conv3d` layer for vision patch embedding.

TICO currently applies `ConvertConv3dToConv2d` to make this operation exportable to an NPU that does not support Conv3D. However, converting the operator itself is not sufficient because the patch embedding path still handles rank-5 tensors.

## Observation

The Qwen3-VL patch embedding Conv3D is a special case:

- The input has already been divided into flattened temporal-spatial patches by the processor.
- `kernel_size` is equal to the full patch shape.
- `stride` is equal to `kernel_size`.
- Each input patch produces exactly one embedding vector.

Therefore, the Conv3D projection

```text
Conv3d:
  weight shape = [embed_dim, channels, temporal_patch_size, patch_size, patch_size]
```

is mathematically equivalent to a Linear projection:

```text
Linear:
  weight shape = [
    embed_dim,
    channels * temporal_patch_size * patch_size * patch_size,
  ]
```

The Conv3D weight can be converted without changing its element order:

```python
linear.weight = conv3d.weight.reshape(embed_dim, patch_dim)
linear.bias = conv3d.bias
```

## Proposal

Change `QuantQwen3VLVisionPatchEmbed` so that it materializes the original Conv3D projection as an equivalent `nn.Linear` layer when the quantization wrapper is created.

The exported patch embedding input should use the following batch-one ABI:

```text
[1, num_patches, patch_dim]
```

where

```text
patch_dim =
    in_channels
    * temporal_patch_size
    * patch_size
    * patch_size
```

The output should preserve the existing Qwen3-VL vision-model contract:

```text
[num_patches, embed_dim]
```

The resulting graph should be

```text
[1, num_patches, patch_dim]
        |
        v
Linear / FullyConnected
        |
        v
[1, num_patches, embed_dim]
        |
        v
reshape or squeeze
        |
        v
[num_patches, embed_dim]
```

Since Qwen3-VL is the only intended use case, no additional generic conversion pass is required. The existing `ConvertConv3dToConv2d` pass should be removed from the default conversion pipeline.

## Quantization compatibility

The existing configuration paths should remain unchanged:

```text
patch_embed.proj.weight
patch_embed.proj.act_in
patch_embed.proj.act_out
```

The converted Linear layer should continue to use:

- Per-channel weight quantization along output-channel axis 0.
- Per-tensor input activation quantization.
- Per-tensor output activation quantization.

This allows existing Qwen3-VL PTQ configurations to work without YAML changes.
